CMSgov/mint-app
# EASI-2398 - Download single model plan CSV

# EASI-2398

## Changes and Description

- Ability to download a single model plan by `modelPlanID` directly from the task list

![Screen Shot 2022-10-11 at 2 35 07 PM](https://user-images.githubusercontent.com/95709965/195172154-ff748616-7062-473a-ab3e-4e61bea29c35.png)

## How to test this change

- Log in as any role and enter a model plan's task list
- On the right side nav, click the link to download that plan as CSV
- Verify correct output in CSV

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
